src/buttons/FullscreenButton/FullscreenButton.js

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import OpenInFullIcon from '@mui/icons-material/OpenInFull';
+import CloseFullscreenIcon from '@mui/icons-material/CloseFullscreen';
+
+export const FullscreenButton = (props) => {
+  const { isFullscreen, setFullscreen } = props;
+
+  return (
+    <>
```

Comment:
Do we need the `<>` wrapper here? It only wraps a single element in both cases.
Maybe we could simply return the result of `isFullscreen ? <CloseFullscreenIcon [...]> : <OpenInFullIcon [...]>`